<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSweep_Dlg</class>
 <widget class="QDialog" name="TimeSweep_Dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/new/prefix4/Ico/time.ico</normaloff>:/new/prefix4/Ico/time.ico</iconset>
  </property>
  <widget class="QPushButton" name="Btn_Measure">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>120</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>测量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Loder">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_IsZero">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>校零</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Quit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>400</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Save">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>190</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_ParaSettings">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参数设置</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_CH1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>通道1</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoRepeatInterval">
    <number>101</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_CH1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_CH2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>通道二</string>
   </property>
   <property name="text">
    <string>通道2</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_CH2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_CH3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>通道3</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_CH3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>checkBox_CH1</tabstop>
  <tabstop>comboBox_CH1</tabstop>
  <tabstop>checkBox_CH2</tabstop>
  <tabstop>comboBox_CH2</tabstop>
  <tabstop>checkBox_CH3</tabstop>
  <tabstop>comboBox_CH3</tabstop>
  <tabstop>Btn_ParaSettings</tabstop>
  <tabstop>Btn_IsZero</tabstop>
  <tabstop>Btn_Measure</tabstop>
  <tabstop>Btn_Save</tabstop>
  <tabstop>Btn_Loder</tabstop>
  <tabstop>Btn_Quit</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
